src/components/sync-connected-sites.tsx

```diff
@@ -301,28 +319,33 @@ const SyncConnectedSitesSection = ( {
 																detail,
 															} );
 														} }
-														disabled={ isAnySitePulling || isAnySitePushing || isOffline }
+														disabled={ isAnySiteSyncing || isOffline }
 													>
 														<Icon icon={ cloudDownload } />
 														{ __( 'Pull' ) }
 													</Button>
 												</Tooltip>
 												<Tooltip
-													text={ getLastSyncTimeWithType(
-														selectedSite.id,
-														connectedSite.id,
-														'push'
-													) }
+													text={
+														isAnySiteSyncing
+															? __(
+																	'Another site is syncing. Please wait for the sync to finish before you push this site.'
```

Comment:
Potentially, we could display one tooltip over Pull and Push buttons to make it less cluttered.

Reply:
Only one is displayed at a time, so it should be good for now. In the future, we could change it to have one label covering the whole area.

Reply:
Sounds good, let's keep as is for now 👍